### PR TITLE
{devel}[system/system] Spack v0.12.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3107.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/0f2f78ff6a9f216b24a8906441fca8b1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
boegel.ugent.be - Darwin UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz, Python 2.7.10
See https://gist.github.com/086fd2b466697abea595fa288a1ef3a6 for a full test report.

**Comment by boegel** [comment]
Returning the favor, cfr. https://github.com/spack/spack/pull/13024

@tgamblin Please review? ;)

File: easybuild/easyconfigs/s/Spack/Spack-0.12.1.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+version = '0.12.1'
+
+homepage = 'https://spack.io/'
+description = """Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific
+ software easy. With Spack, you can build a package with multiple versions, configurations, platforms, and compilers,
+ and all of these builds can coexist on the same machine."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/LLNL/spack/archive/']
```
Comment (adamjstewart) [inline easybuild/easyconfigs/s/Spack/Spack-0.12.1.eb]:
```suggestion
source_urls = ['https://github.com/spack/spack/archive/']
```
It should redirect, but this was updated a while ago so might as well be consistent.

**Comment by smoors** [comment]
rejected :smile:

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**